src/freenas/root/.zshrc

```diff
@@ -1,9 +1,14 @@
 
 HISTFILE=~/.zsh-histfile
 SAVEHIST=1000
+PROMPT="%n@%m# "
 
 setopt APPEND_HISTORY
 
+bindkey "^[[F" end-of-line
+bindkey "^[[H" beginning-of-line
+bindkey "^[[3~" delete-char
+
```

Comment:
I also suggest to include this:

```
# Alt+Left/Right
bindkey '\e[1;5D' backward-word
bindkey '\e[1;5C' forward-word
# Alt+Delete
bindkey '\e\e[3~' delete-word
# Ctrl+Backspace
bindkey '^H' backward-delete-word
```

Reply:
`set -o emacs` cancels it (but running `bindkey` again brings it back)

Emacs mode probably offers combinations for these actions so nevermind then